pkg/instance/instance.go

```diff
@@ -455,6 +414,25 @@ func (inst *inst) testRepro() ([]byte, error) {
 	return out, err
 }
 
+func (inst *inst) csourceOptions() (csource.Options, error) {
+	if len(inst.reproSyz) == 0 {
+		return csource.Options{}, nil
```

Comment:
Does it make any difference if we use an empty `csource.Options{}` or `csource.DefaultOpts(cfg)` here?

Reply:
Changed to panic with a comment.
PTAL